packages/subgraph/manifest/data/arbitrum.json

```diff
@@ -10,9 +10,9 @@
       }
     ],
     "Plugin": {
-      "name": "MyPlugin",
-      "address": null,
-      "startBlock": 7727664
+      "name": "multisig.plugin.dao.eth",
```

Comment:
confirmed that Claudia and I did things the other way: XXPluginRepo rather than ENS

Reply:
This is supposed to get fixed in https://aragonassociation.atlassian.net/browse/OS-1148

Reply:
Fixed it now according to your comment and the task above